src/mcp_gateway/audit/logger.py

```
@@ -0,0 +1,74 @@
+    def log(self, *, ev: str, level: Literal["INFO", "DEBUG"] = "INFO", **fields: Any) -> None:
+        # ログレベルによるフィルタリング
+        if self.level == "INFO" and level == "DEBUG":
+            return
+
+        # 予約キーのチェック
+        reserved = {"ts", "ev", "level"}
+        conflicted = reserved & fields.keys()
+        if conflicted:
+            raise ValueError(f"reserved audit field(s): {', '.join(sorted(conflicted))}")
+
+        # タイムスタンプ（マイクロ秒精度）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)